presto-hive/src/main/java/com/facebook/presto/hive/HiveWriteUtils.java

```diff
@@ -226,6 +234,9 @@ public static Object getField(Type type, Block block, int position)
         if (DoubleType.DOUBLE.equals(type)) {
             return type.getDouble(block, position);
         }
+        if (RealType.REAL.equals(type)) {
```

Comment:
Done in 522b782836a6a67629279b5b81a3015046359751.
But can you explain what's the consistency? I followed here the logic of "bigger type of the same family comes first". I kind of mirrored BIGINT, INTEGER etc. order.
Alphabetical order does not make sense, especially if you have a look on integer types.